src/commands/send-request.js

```diff
@@ -75,6 +76,60 @@ const parseHttpFile = async (filePath, variables, rawMode) => {
 
 const removeComments = (input) => input.replace(/(\r?\n|^)(#|\/\/).*$/gm, "");
```

Comment:
Not related to scope of this pr, but ensure that comments are mentioned in --help command (help.js)